samples/ftd_configuration/smart_agent_connection.yaml

```diff
@@ -0,0 +1,41 @@
+- hosts: all
```

Comment:
Let's merge this playbook with `smart_license.yml` as they are essentially doing the same things.

Reply:
Done